.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+###### Description
+
+
+###### System Info
+macOS 10.
+Xcode 8.
+
+###### Verification
+- [ ] Have you checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
+- [ ] Have you referenced relating tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL?
+- [ ] Have you checked your Portfile with `port lint`?
+- [ ] Have you tried existing tests with `sudo port test`?
+- [ ] Have you tried a full install with `sudo port install`?
+- [ ] Have you tested basic functionality of all binary files?
```

Comment:
I would add: "(delete if this is not installing any binary files)".

This reminded me that we should remind users to revbump dependent ports in case it's a library, but again I have no clue about the best way to do it.